### ISSUE TITLE
REST API: Endpoints for static files inside a project

### ISSUE THREAD

**Issue by stefanbschneider** [sonata-nfv/tng-sdk-project#83]
Use Flask's `send_from_directory` to serve all files within a project directory: https://stackoverflow.com/questions/20646822/how-to-serve-static-files-in-flask

There should be a new endpoint `/v1/projects/{project_uuid}/{file_path_and_name}` to access each file in a project with a given UUID.

`{file_path_and_name}` should be path to the file inside the project directory and the file name, e.g., `tango_vnfd0.yml` or `vnfds/tango/vnfd0.yml`. This should be handled directly by Flask and should not need any additional effort.

This can then be used to display the files in the SDK portal frontend or to download them.

**Comment by stefanbschneider** [comment]
Once, we have this, we can update the response to POST `/project` to include the endpoint URL for all generated files.

**Comment by zulfiqarjunejo** [comment]
`# url model for static file
url_model = api_v1.model("Url", {
	"project_uuid": fields.String(description="Project UUID", required=True),
	"file_name": fields.String(description="file name", required=True),
	"url": fields.String(description="url for the requested file"),
	"error_msg": fields.String(description="error message")
})`

`@api_v1.route("/projects/<string:project_uuid>/<string:file_name>")
class Projects(Resource):
	#@api_v1.marshal_with(url_model)
	def get(self, project_uuid, file_name):
		print(project_uuid)
		print(file_name)
		print(app.root_path)
		print(app.instance_path)
		return {"project_uuid": project_uuid, "file_name": file_name, "error_msg": send_from_directory(project_uuid, file_name)}`

**Comment by zulfiqarjunejo** [comment]
This was just an experiment so I didn't really care about sending the file as `error_msg`.

**Comment by stefanbschneider** [comment]
What was the error?

**Comment by stefanbschneider** [comment]
This works: https://github.com/stefanbschneider/tng-sdk-project/blob/dev/static/src/tngsdk/rest.py#L62

```python
@app.route('/static/<path:path>')
def send_js(path):
    static_dir = os.path.join(os.getcwd(), 'example-project')
    log.debug("Serving static files from: {}".format(static_dir))
return send_from_directory(static_dir, path)
```

GET to `localhost:5098/static/osm_nsd.yml` returns the file.

Problem was apparently that the directory path was not set properly. Simply using a string doesn't work.

**Comment by stefanbschneider** [comment]
* fix CI failure
* add example in readme
* add doc string for Swagger
* add tavern tests

**Comment by zulfiqarjunejo** [comment]
[add example in readme] -> Done
 
[add doc string for Swagger] -> Done

**Comment by stefanbschneider** [comment]
Closed by #90
